pytest_fixtures/core/sat_cap_factory.py

```diff
@@ -292,10 +301,10 @@ def parametrized_enrolled_sat(
     Broker(hosts=[new_sat]).checkin()
 
 
-def get_deploy_args(request):
-    """Get deploy arguments for Satellite base OS deployment. Should not be used for Capsule."""
+def get_sat_deploy_args(request):
```

Comment:
How if we move these to their own classes in `robottelo/hosts.py` module as class methods?

Once moved, We/Anywhere these arguments could be used.